packages/twenty-front/src/modules/object-record/record-table/record-table-body/components/RecordTableNoRecordGroupBodyEffect.tsx

```diff
@@ -97,8 +102,22 @@ export const RecordTableNoRecordGroupBodyEffect = () => {
 
   useEffect(() => {
     (async () => {
-      if (!isFetchingMoreObjects && tableLastRowVisible && hasNextPage) {
-        await fetchMoreDebouncedIfRequested();
+      if (
+        !isFetchingMoreObjects &&
+        tableLastRowVisible &&
+        hasNextPage &&
+        !encounteredUnrecoverableError
+      ) {
+        const result = await fetchMoreDebouncedIfRequested();
+
+        const isForbidden =
+          result?.error?.graphQLErrors.some(
+            (e) => e.extensions?.code === 'FORBIDDEN',
```

Comment:
Agree with the strat, I think we could do it for all errors (not only FORBIDDEN)
Thanks for fixing this one!

Reply:
ok, LGTM!